Greenline/ConfigExtractor.cs

```diff
@@ -55,8 +55,22 @@ public static void TryParseConfig(TypeDefinition type, out string? config)
 
     private static string Decrypt(string input, string key)
     {
-        string first = FromBase64(input);
-        return FromBase64(Xor(first, key));
+        bool Base64String = IsBase64String(input);
+        if (Base64String)
+        {
+            string first = FromBase64(input);
+            return FromBase64(Xor(first, key));
+        }
+        else
+        {
+            return input;
+        }
+    }
+
+    public static bool IsBase64String(string base64)
+    {
+        Span<byte> buffer = new Span<byte>(new byte[base64.Length]);
+        return Convert.TryFromBase64String(base64, buffer, out int bytesParsed);
```

Comment:
Could you please change `out int bytesParsed` to `out int _` since we discard that value to comply with code style.

Reply:
oh Sure.